onboarding_auth/onboardingAuthPresentation/src/main/java/com/alexafit/onboardingAuthPresentation/register/RegisterScreen.kt

```diff
@@ -0,0 +1,17 @@
+package com.alexafit.onboardingAuthPresentation.register
+
+import androidx.compose.material.ScaffoldState
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.alexafit.onboardingAuthPresentation.login.LoginViewModel
+import com.alexafit.onboardingAuthPresentation.login.event.LoginEvent
+
+@Composable
+fun RegisterScreen(
+    scaffoldState: ScaffoldState,
+    onEventClick: (LoginEvent) -> Unit,
+    viewModel: LoginViewModel = hiltViewModel()
```

Comment:
Is there a reason you use `LoginViewModel` here? Probably just copy-paste for now I guess?

Reply:
Going to change it in the next PR. It was just copy pasted at the moment